augeas-mode.el

```diff
@@ -74,7 +74,7 @@ Argument IGNORE is not used here."
 (defvar augeas-mode-syntax-table
   (let ((st (make-syntax-table)))
       (modify-syntax-entry ?_ "w"    st)
-      (modify-syntax-entry ?\" "_"   st)
+      (modify-syntax-entry ?\" "."   st)
```

Comment:
What purpose has this change? For me it breaks the following code:
```
let filter = incl "/etc/foo"
           . incl "/etc/bar"
           . Util.stdexcl
```

The highlighting style changes halfway through the strings here. The beginning has `font-lock-string-name-face` and the end has `font-lock-variable-name-face`.